### PR TITLE
VCDI credential offer demo

### PR DESCRIPTION
**Pull request by ianco**
Demo of generating the VCDI credential offer attachment.  There were some issues with the schema that I fixed.

(not production code but an example of what needs to be done)

If you run the alice/faber demo faber will send the offer and alice will receive it.  Still TBD is sending the appropriate credential request (alice to faber) and then the credential issue (faber to alice)

### CODE REVIEW

**Review by gvelez17 — APPROVED** [review]
thanks!

File: aries_cloudagent/indy/models/cred_abstract.py [hunk 1]
```diff
@@ -331,22 +331,22 @@ class Meta:
         model_class = VCDICredAbstract
         unknown = EXCLUDE
 
-        data_model_versions_supported = fields.List(
-            fields.Str(), required=True, metadata={"description": "", "example": ""}
-        )
```
Comment (gvelez17) [inline aries_cloudagent/indy/models/cred_abstract.py:336]: @tra371 @supersonicwisd1 so this means, these are not supposed to be metadata, they are supposed to be in the credential?

File: aries_cloudagent/protocols/issue_credential/v2_0/formats/vc_di/handler.py [hunk 2]
```diff
@@ -253,7 +253,35 @@ async def _create():
         if not cred_offer:
             cred_offer = await _create()
 
-        return self.get_format_data(CRED_20_OFFER, cred_offer)
+        vcdi_cred_offer = {
```
Comment (gvelez17) [inline aries_cloudagent/protocols/issue_credential/v2_0/formats/vc_di/handler.py:256]: thank you @ianco !  for production do you think we should move this into the issuer, perhaps in a VcdiIssuer analogous to IndyIssuer, or an alternate create_credential_offer function?

Comment (ianco) [reply aries_cloudagent/protocols/issue_credential/v2_0/formats/vc_di/handler.py:256]: After some thought I think this should be in the vc_di handler class and you shouldn't add new methods to the AnonCredsIssuer (unless they are direcrtly related to a method you need to call in anoncreds-rs).  In this case I think the same cred offer is used and it's just a matter of formatting the attachment for the credential exchange.

**Pull request merged**